src/lib/zxing/index.ts

```diff
@@ -0,0 +1,27 @@
+import { BrowserQRCodeReader, IScannerControls } from '@zxing/browser'
```

Comment:
```suggestion
import type { IScannerControls } from '@zxing/browser'
```

Remove `BrowserQRCodeReader`. Remember, the lib must lazy loaded.

Reply:
Use `import type` if you need it only for TS. It will not be included in the bundle.

```ts
import type { BrowserQRCodeReader , IScannerControls } from '@zxing/browser'
```